drivers/sensor/maxim/ds18b20/ds18b20.c

```diff
@@ -5,12 +5,10 @@
  */
 
 /**
- * Driver for DS18B20 1-Wire temperature sensors
- * A datasheet is available at:
- * https://datasheets.maximintegrated.com/en/ds/DS18B20.pdf
- *
- * Driver also support the older DS18S20 1-Wire temperature sensors.
+ * Driver for DS18B20 and DS18S20 1-Wire temperature sensors
+ * A datasheets is available at:
```

Comment:
```suggestion
 * A datasheet is available at:
```

Reply:
Good catch! As now both datasheets are listed below what do you think about:
` Datasheets for the compatible sensors are available at:`